dashboard/test/ui/features/i18n.feature

```diff
@@ -0,0 +1,44 @@
+Feature: See Hour of Code and Frozen tutorial in Spanish, Portuguese, and Arabic
+
+Background:
+  Given I am on "http://studio.code.org/reset_session"
+
+Scenario: HoC tutorial in Spanish
+  Given I am on "http://studio.code.org/s/20-hour/stage/2/puzzle/16/lang/es"
+  And I rotate to landscape
+  And I wait to see ".modal"
+  Then element ".dialog-title" has text "Puzzle 16 de 20"
+  Then element ".modal-content p:nth-child(2)" has text "¡Yo querer girasol! Usa un bloque \"si\" para llevarme allí con la menor cantidad posible de bloques."
+  And element "#prompt" has text  "¡Yo querer girasol! Usa un bloque \"si\" para llevarme allí con la menor cantidad posible de bloques."
```

Comment:
Can we pull the actual value from the i18n ymls so this test doesn't break if the translations are updated?